File: solutions/4179-largest-even-number/largest-even-number.cs
public class Solution {
    public string LargestEven(string s) {
        int ind = -1;

        for (int i = s.Length - 1; i >= 0; i--) {
            if (s[i] == '2') {
                ind = i;
                break;
            }
        }

        return s.Substring(0, ind + 1);
    }
}

File: solutions/4179-largest-even-number/largest-even-number.js
var largestEven = function(s) {
    let ind = -1;

    for (let i = s.length - 1; i >= 0; i--) {
        if (s[i] === '2') {
            ind = i;
            break;
        }
    }

    return s.slice(0, ind + 1);
};